.github/workflows/ci.yml

```diff
@@ -36,6 +36,11 @@ jobs:
             redis-version: 'latest'
             python-version: '3.9'
 
+          # Django 4.2 and python 3.12 with latest redis
```

Comment:
this is not needed, please also add python 3.12 to the matrix

Reply:
py38-dj42-redislatest    
py39-dj42-redislatest    
py310-dj42-redislatest   
py310-dj50-redislatest   
py310-dj51-redislatest   
py311-dj42-redislatest   
py311-dj50-redislatest   
py311-dj51-redislatest   
py312-dj42-redislatest   
py312-dj50-redislatest   
py312-dj51-redislatest   
py311-dj51-redismaster   
py311-djmain-redislatest 
py311-djmain-redismaster

This is the tox matrix now that matches the ci matrix.